### ISSUE TITLE
When installing a low-version apk, the exception cannot be caught, and 'catch' needs to be added. At the same time, I hope to increase the '-d' parameter, such as 'install -d -r'

### ISSUE THREAD

**Issue by BabyCY** [DeviceFarmer/adbkit#239]
![image](https://user-images.githubusercontent.com/5268512/150295665-f6f70f24-529b-4741-9954-8f86564dc225.png)
![image](https://user-images.githubusercontent.com/5268512/150295837-317fbc75-6c52-4c30-bd0f-29e742fb1f3c.png)
![image](https://user-images.githubusercontent.com/5268512/150296446-80f55b9e-1c6b-440e-b94c-b3173a885d03.png)

**Comment by koral--** [comment]
I don't understand what is your expectation?
Do you believe that some code should be added? If so please submit a PR.

**Comment by NoahAndrews** [comment]
If you submit a PR, please make sure that `-d` is an opt-in behavior, not the default.

**Comment by BabyCY** [comment]
> 

Because I didn't study it too deeply, I'm not sure if this modification will have any side effects, so I just tried a repair idea and won't submit a PR. I hope someone who understands the principle better can fix it based on these suggestions, or have a better solution.

**Comment by BabyCY** [comment]
> I don't understand what is your expectation? Do you believe that some code should be added? If so please submit a PR.

@koral-- Because I didn't study it too deeply, I'm not sure if this modification will have any side effects, so I just tried a repair idea and won't submit a PR. I hope someone who understands the principle better can fix it based on these suggestions, or have a better solution.

**Comment by BabyCY** [comment]
> If you submit a PR, please make sure that `-d` is an opt-in behavior, not the default.

@NoahAndrews Yes, I hope so too, many parameters cannot be customized at present. But I don't know how to modify it, so I won't submit a PR.
